package.json

```
@@ -10,10 +10,13 @@
     "gen-icon-bindings": "node genbindings.js"
   },
   "keywords": [
-    "BuckleScript"
+    "BuckleScript",
+    "Rescript",
+    "FontAwesome",
+    "Font Awesome"
   ],
-  "dependencies": {
-    "reason-react": "^0.9.1"
+  "peerDependencies": {
+    "@rescript/react": "^0.10.1"
```

while we're at it let's add the peerDependency on bs-platform

```suggestion
    "@rescript/react": "^0.10.1",
    "bs-platform": "^8.4 || ^9"
```

---

Not sure I understand why this would be included in peerDependencies? This would have to be a devDependency in the project that is using the binding correct?

---

I think that's my mistake -- this seems fine the way you put it